package.json

```diff
@@ -65,11 +66,15 @@
         "includeFailureMsg": true
     },
     "dependencies": {
+        "@fortawesome/fontawesome": "^1.1.8",
+        "@fortawesome/fontawesome-free-solid": "^5.0.13",
         "@types/sprintf-js": "^1.1.0",
         "axios": "^0.17.1",
         "css-element-queries": "^0.4.0",
         "css-loader": "^0.28.7",
         "enquire.js": "^2.1.6",
+        "froala-editor": "^2.8.4",
```

Comment:
I believe we should only see this dependency and not the others.

Reply:
@fortawesome/fontawesome": "^1.1.8 and @fortawesome/fontawesome-free-solid": "^5.0.13 aren't mandatory if you use font-awesome 4... you'd still have to install font-awesome 4 though.

froala needs jquery to work.  I also need jQuery in vue-froala.ts